### PR TITLE
feat(tsf): enable language bar button

### PR DESCRIPTION
**Pull request by nameoverflow**

### CODE REVIEW

File: WeaselTSF/LanguageBar.cpp [hunk 1]
```diff
@@ -137,7 +141,7 @@ STDAPI CLangBarItemButton::Show(BOOL fShow)
 
 STDAPI CLangBarItemButton::GetTooltipString(BSTR *pbstrToolTip)
 {
-	*pbstrToolTip = SysAllocString(L"左鍵切換模式，右鍵打開菜單");
+	*pbstrToolTip = SysAllocString(L"右鍵打開菜單");
```
Comment (Prcuvu) [inline WeaselTSF/LanguageBar.cpp]: It would be better to implement `ascii_mode` switch on language bar icon click.

File: weasel.sln [hunk 2]
```diff
@@ -1,13 +1,14 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 14
-VisualStudioVersion = 14.0.25420.1
+# Visual Studio 15
```
Comment (lotem) [inline weasel.sln]: Why is this change?

Comment (nameoverflow) [reply weasel.sln]: Auto generated by IDE

Comment (nameoverflow) [reply weasel.sln]: Reverted.

**Pull request merged**